markdowns/20_Using_Haystack_with_REST_API.md

```diff
@@ -0,0 +1,312 @@
+        --header 'content-type: multipart/form-data' \
+        --form files=@article_txt_countries_and_capitals/0_Minsk.txt \
+        --form meta=null
+   ```
+
+   This method is not the best one if you have multiple files to upload. That's because you need to replace file names in the request by hand. Instead, you can run a command that takes all *.TXT* files in the *article_txt_countries_and_capitals* folder and sends a POST request to index each file:   
+
+   ```bash
+   find ./article_txt_countries_and_capitals -name '*.txt' -exec \
+        curl --request POST \
+             --url http://127.0.0.1:8000/file-upload \
+             --header 'accept: application/json' \
+             --header 'content-type: multipart/form-data' \
+             --form files="@{}" \
+             --form meta=null \;
+   ```
+
+## Asking a Question
+
+That's it, the application is ready! Send another POST request to retrieve documents about _"climate in Scandinavia"_: 
+
+```bash
+curl --request POST \
+     --url http://127.0.0.1:8000/query \
+     --header 'accept: application/json' \
+     --header 'content-type: application/json' \
+     --data '{
+     "query": "climate in Scandinavia"
+     }'
+```
+
+As a response, you will get a `QueryResponse` object consisting of `query`, `answers`, and `documents`. Documents related to your query will be under the `documents` attribute of the object.
+
+```python
+{
+  "query": "climate in Scandinavia",
+  "answers": [],
+  "documents": [
+    {
+      "id": "52937ad257317032b9aed9750b5fcbb7",
+      "content": "Even though temperature patterns differ between north and south, the summer climate is surprisingly similar all through the entire country in spite of the large latitudinal differences. This is due to the south's being surrounded by a greater mass of water, with the wider Baltic Sea and the Atlantic air passing over lowland areas from the south-west. ...",
+      "content_type": "text",
+      "meta": {
+        "_split_id": 7,
+        "name": "43_Sweden.txt"
+      },
+      "score": 0.569930352925387
+    },
+    ...
+  ]
+}
+```  
+
+Congratulations! You have created a proper search system that runs using Haystack REST API.
+
+## About us
+
+This [Haystack](https://github.com/deepset-ai/haystack/) tutorial was made with love by [deepset](https://deepset.ai/) in Berlin, Germany
+
+We bring NLP to the industry via open source!  
+Our focus: Industry specific language models & large scale QA systems.  
+  
+Some of our other work: 
+- [German BERT](https://deepset.ai/german-bert)
+- [GermanQuAD and GermanDPR](https://deepset.ai/germanquad)
+- [FARM](https://github.com/deepset-ai/FARM)
+
+Get in touch:
+[Twitter](https://twitter.com/deepset_ai) | [LinkedIn](https://www.linkedin.com/company/deepset-ai/) | [Discord](https://haystack.deepset.ai/community/join) | [GitHub Discussions](https://github.com/deepset-ai/haystack/discussions) | [Website](https://deepset.ai)
```

Comment:
Change https://haystack.deepset.ai/community/join to https://haystack.deepset.ai/community